github/pull_request.rb

```diff
@@ -128,7 +137,7 @@ def check_commits_style
   end
 
   def add_issue_links
-    if new? && issue_numbers.any?
+    if new? && issue_numbers.any? && !cherry_pick?
```

Comment:
I think this is not needed. If the PR does include both issue numbers and CPs then it's fine to link those that are included. If there aren't any, it's a noop anyway. It doesn't hurt either though.

Reply:
So, are you saying it's fine to have cherry-pick PRs linked to the redmine issues (well, just the first commit's issue)? I find the behavior annoying and inconsistent. Maybe if it linked *all* the issues in a cherry-pick PR, it'd be fine, but it's just the first one. I'd rather just not have the extra links.

Reply:
Also fair. Since I usually don't use those links, I'm fine with whatever you decide.

Reply:
@ekohl oh, right. :) I do want to disable adding links to redmine also @zjhuntin.